### ISSUE TITLE
Problem with parsing item names from RU client. The presence of the character "*" in the names.

### ISSUE THREAD

**Issue by niktoxa** [recursivetree/seat-treelib#3]
```
{
    "warning": true,
    "_debug_parser": null,
    "_debug_text": "Ashimmu Blueprint*\t\t\t0,01 \u043c^3\r\nAstro-Inertial Compact Missile Guidance Computer*\t39\t\t195 \u043c^3\r\nAugmentation Decryptor*\t3\t\t0,30 \u043c^3\r\nBomb Launcher I*\t1\t\t50 \u043c^3\r\nBroken Drone Transceiver*\t1\t\t0,01 \u043c^3\r\nCaldari Navy Cloaking Device*\t2\t\t200 \u043c^3\r\nCaldari Navy Inferno Heavy Missile*\t34\u00a0586\t\t1\u00a0037,58 \u043c^3\r\nCap Recharger II*\t4\t\t20 \u043c^3\r\nCharred Micro Circuit*\t1\u00a0018\t\t10,18 \u043c^3\r\nCore B-Type Thermal Armor Hardener*\t1\t\t5 \u043c^3\r\nDark Blood Reactor Control Unit*\t7\t\t35 \u043c^3\r\nDatacore - Amarrian Starship Engineering*\t18\t\t1,80 \u043c^3\r\nDatacore - Electromagnetic Physics*\t1\t\t0,10 \u043c^3",
    "items": []
}
```
- I'm trying to copy the content of the inventory window list and put the content in a plugin form.
- Copy pasted from the EVE inventory window

**Comment by recursivetree** [comment]
fixed with 1.3.1

**Comment by niktoxa** [comment]
Thx a lot!! Great job!
